selfdrive/car/toyota/interface.py

```diff
@@ -145,10 +145,10 @@ def get_params(candidate, fingerprint, vin=""):
       ret.safetyParam = 100
       ret.wheelbase = 2.78
       ret.steerRatio = 16.0
-      tire_stiffness_factor = 0.444 # not optimized yet
+      tire_stiffness_factor = 0.7933 # from overtime live tuning result
       ret.mass = 4607 * CV.LB_TO_KG + std_cargo #mean between normal and hybrid limited
-      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.6], [0.05]]
-      ret.lateralTuning.pid.kf = 0.00006
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.18], [0.0075]] # from Phil Lee's PR
```

Comment:
Ki seems way too low (as is, it does nothing). I propose to use the same tuning as the Avalon here, `[0.17], [0.03]`.

Kf at 0.00012 is fine.

Reply:
I will try with your suggested values and get back here. Thanks Riccardo!

Reply:
[0.17], [0.03] is better than [0.6], [0.05]. But still it has jerk movement with [0.17], [0.03] and [0.17], [0.025] under 35mph. I will try with [0.17], [0.02] today and report it again.

Reply:
@rbiasini, [0.17], [0.02] also has same jerk movement. So I updated with old stable parameters,  which is [0.18], [0.015]. I used this parameters more than 4 months without any issues. Please review and merge this request.